testing/dumux.sh

```diff
@@ -1,20 +1,21 @@
 # this tests dumux as external package which provides
 # a Python package outside the dune namespace
 
-# dumux Python bindings currently only work with shared libs
-export DUNE_CMAKE_FLAGS=-DBUILD_SHARED_LIBS=ON
 dumux_url="https://git.iws.uni-stuttgart.de/dumux-repositories/dumux.git"
 
 # setup external venv and install the prepared packages needed
 # for dumux
 python3 -m venv dune-env
 source dune-env/bin/activate
-pip install mpi4py
-pip install --pre --find-links file://$PWD/../dist dune.common dune.grid dune.geometry dune.localfunctions dune.istl
+pip install mpi4py requests
+pip install --pre --find-links file://$PWD/../dist dune.common dune.grid dune.geometry dune.localfunctions dune.istl dune.alugrid
```

Comment:
Alugrid is not used in the Python tests and should definitely not be required (only optional). What was the error message here?

Reply:
I didn't check the message. Perhaps best to remove it again and retest before we merge?

Reply:
```suggestion
pip install --pre --find-links file://$PWD/../dist dune.common dune.grid dune.geometry dune.localfunctions dune.istl
```